ddtrace/internal/remoteconfig/client.py

```diff
@@ -370,17 +378,22 @@ def _process_response(self, data):
                 log.debug("Disable configuration: %s", target)
                 callback_action = False
 
-            callback = self._products[config.product_name]
-
-            try:
-                callback(config, callback_action)
-            except Exception:
-                log.debug("error while removing product %s config %r", config.product_name, config)
-                continue
+            callback = self._products.get(config.product_name)
```

Comment:
we check if the product exists [here](https://github.com/DataDog/dd-trace-py/pull/5032/files#diff-7f727a2a181fcfac54ea16cde6b40dc992e8b49d5c898b85400ee361c22889c8R342), we don't need `.get`

I think `if callback` it isn't needed

Reply:
Code updated :+1:

Reply:
Code updated :+1: